styles.css

```diff
@@ -256,7 +263,5 @@ div.transbox:hover{
     width: 240px;
   }
 
-  article {
-    margin-left: auto;
```

Comment:
I think this is what made it stop moving left when it was small, I'm guessing it was causing problems